### PR TITLE
Turn off FutureLearn update jobs on a specified date

### PR DESCRIPTION
**Pull request by dcorking**
## Status

* Current Status: Ready for tech review
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2282

* https://github.com/NCCE/terraform/pull/179 deploys the env vars

## Review progress:

- [x] Tech review completed

## What's changed?

When NCCE2 starts, turn off
- FutureLearn::CourseRunsJob
- FutureLearn::UpdateOrganisationMembershipsJob

## Steps to perform after deploying to production

Set
```
FLAG_NCCE2=on
NCCE2_START_DATE='2023-04-02'
```
in production when you are ready

### CODE REVIEW

File: .env.defaults [hunk 1]
```diff
@@ -36,8 +36,14 @@ FL_PARTNERS_CONSUMER_SECRET=secret
 FL_PARTNERS_URL=https://lticourses-api.sandbox.futurelearn.com # The sandbox is currently disabled
 FL_PARTNERS_IGNORED_COURSE_UUIDS=
 
+FLAG_NCCE2=on
+NCCE2_START_DATE='2023-04-02'
+
 MYLEARNING_DASHBOARD_URL=https://mylearning.stem.org.uk/my/
 
+FLAG_NCCE2_TEST=off
+FLAG_NCCE2_LIVE=off
```
Comment (danhalson) [inline .env.defaults:45]: Just curious if these were intended to be left in, as they don't seem to be used as part of these changes?

**Review by danhalson — APPROVED** [review]
LGTM (left one query, but approved anyway as it's not a blocker)

**Review by danhalson — APPROVED** [review]
LGTM

**Comment by dcorking** [comment]
@mjholmes are you OK with the basic principle of this PR? if so we can deploy it on Monday or Tuesday, and it will take effect on the morning of Sunday 2nd April? We can discuss at standup.

**Comment by mjholmes** [comment]
> @mjholmes are you OK with the basic principle of this PR?

Yep, fine with this thanks.

**Pull request merged**